components/ide-proxy/Dockerfile

```diff
@@ -18,9 +18,11 @@ RUN for FILE in `ls /bin/gitpod-local-companion*`;do \
   gzip -v -f -9 -k "$FILE"; \
 done
 
+RUN mkdir -p static/code
+RUN wget https://raw.githubusercontent.com/gitpod-io/openvsx.org.json/main/marketplace.json -O static/code/marketplace.json
 
 FROM caddy/caddy:2.4.6-alpine
 
 COPY conf/Caddyfile /etc/caddy/Caddyfile
-COPY static /www/
-COPY --from=compress /bin /www/static/bin
\ No newline at end of file
+COPY --from=compress static /www/
```

Comment:
This broken the original behavior, which makes the ide logo not exist

Reply:
![image](https://user-images.githubusercontent.com/20944364/179223943-21e7db58-6078-451d-9b10-020d44a9162c.png)

Reply:
Thanks for noticing! I tried this but probably had them cached so it didn't show up while developing. I'll fix it.